test/assertions.rb

```diff
@@ -1,6 +1,5 @@
 module Assertions
   def assert_method_visibility(object, method_name, visiblity)
-    method_key = method_name.to_sym
-    assert object.send("#{visiblity}_methods").include?(method_key), "#{method_name} is not #{visiblity}"
+    assert object.send("#{visiblity}_methods").include?(method_name), "#{method_name} is not #{visiblity}"
```

Comment:
Is this possible, because `Object#public_methods`, etc, returns an array of strings in Ruby > v1.8; but returns an array of symbols in Ruby v1.8? If so, it would be good to explain that in the commit note.

Reply:
Yes. And existing callers of `assert_method_visibility` only ever pass the `method_name` as a symbol.

However, on second thoughts, instead of removing the `to_sym` conversion, I've just inlined the temp `method_key`.

Reply:
Thanks. I've convinced myself that not calling `to_sym` is safe here, so I've made that change.